cgi-bin/DW/Media.pm

```diff
@@ -110,9 +110,15 @@ sub upload_media {
 
     # set the security
     my $sec = $opts{security} || 'public';
+    $sec = 'usemask' if $sec =~ /^(?:friends|access)$/;
```

Comment:
I'm pretty sure I saw at least one place where "usemask" was being used in lieu of "access" and no allowmask was passed in. Do I need to find it again and change it, or can I do this to protect against it on general principle?